DoSomething/rogue
# Reviewing reportback items bug

#### What's this PR do?
- If a post cannot be found by `event_id`, then it is in the old data schema. Send this to the `updateReportbackItems` function and deal with following old logic (when we sent reviewed reportback items to `/items`). 
- Before returning to the controller, remove any Reportback Items objects from `$reviewedPosts` array and put in it's own array, `$reportbackItems`.
- If `$reviewedPosts` is empty, this means all the reviewed items are Reportback Item objects.  Return only if all reviewed items are reportback item objects. Then we will send all to the Reportback Items Controller. 
- If `$reviewedPosts` is not empty, this means there are at least one reviewed items that is a Post object. The `$reviewedPost` array omits reportback item objects so the Post transformer doesn't break and Phoenix puts in `dosomething_rogue_failed_task_log`.  

#### How should this be reviewed?
- Find a `rogue_event_id` in Pheonix's `dosomething_rogue_reportbacks` that doesn't exist in Rogue's `events` table but _does_ exist in the `reportback_items` table. 
- Find the corresponding `fid` and set to `pending`.
- Go to the campaign and review the fid. 
- Check to make sure the corresponding reportback item's status in Rogue is updated.
